pkg/daemon/ceph/rgw/objectstore.go

```diff
@@ -259,6 +259,13 @@ func createSimilarPools(context *Context, pools []string, poolSpec model.Pool) e
 			poolSpec.FailureDomain, poolSpec.CrushRoot); err != nil {
 			return fmt.Errorf("failed to create erasure code profile for object store %s: %+v", context.Name, err)
 		}
+
+		// attempt to disable EC overwrites on the new pool
+		if err := ceph.SetPoolProperty(context.context, context.ClusterName, poolSpec.Name, "allow_ec_overwrites", "false"); err != nil {
```

Comment:
This would actually need to be called on each pool in the `for` loop below, but this is a moot point with the previous suggestion.